webapp/src/app/pages/components/knowledge-graph/cytoscape/cytoscape.component.ts

```
@@ -419,8 +450,14 @@
         this.nodeSelected = false;
         if (eventTarget !== this.cy) {
           if (eventTarget.isNode()) {
+            // If there is a previously selected node, reset its border style.
+            if (this.selectedNodeElement) {
+              this.selectedNodeElement.style('border-color', '');
+              this.selectedNodeElement.style('border-width', '');
+            }
             console.log('weight: ' + eventTarget.data('weight'));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)